### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by brihsaauce** [notgoodusename/OsirisAndExtra#320]
pixelsurf and minijump addded into the chea

**Comment by brihsaauce** [comment]
what about pixelsurf?

**Comment by Xsintashi** [comment]
> minijump is detected

how minijump works cuz i wanna implement that shit into my paste but idk how it even works

**Comment by brihsaauce** [comment]
> > minijump is detected
> 
> how minijump works cuz i wanna implement that shit into my paste but idk how it even works

yeah man same idk why this guy is sayin "i dont see any use" its a pretty cool movement feature so i dont get it bro

**Comment by brihsaauce** [comment]
do u know anyone who knows how to put pixelsurf into a cheat?

**Comment by Xsintashi** [comment]
> bruh its detected. When i played with osiris back in the day when it had mini jump it was detected
Gamesense has something like this. If some functions are detected, they are marked with yellow text.

**Comment by brihsaauce** [comment]
okay but do u know anyone who any1 who knows how to pixelsurf into a cheat
